Remove unused  setupegg.py

kinow opened the pull request:
Hi,

Looks like the `setuptools` module is no longer used, but instead a command is executed calling `setup.py`. This PR simply removes the unused import and the related comment :+1:

kinow commented:
ps: quite sure the test that failed is not related and just needs a kick on Travis

minrk commented:
If you'd like, you can just remove `setupegg.py`. It's from an ancient way of avoiding setuptools that we don't anymore.

kinow commented:
> If you'd like, you can just remove `setupegg.py`. It's from an ancient way of avoiding setuptools that we don't use anymore.

Oh, that's even easier. Dropped previous commit, and added a new one removing `setupegg.py`. Updated pull request title as well. Thanks!

minrk commented:
Thanks!
